managing-os/templates/archlinux.md

```diff
@@ -213,11 +216,19 @@ Main maintainer of this template is [Olivier Médoc](mailto:o_medoc@yahoo.fr).
 
         *   Screen '**Get sources**' wants to download additional packages needed for the choosen plugin/s.
         
-            *   Select '**No**' Press '**Enter**'
```

Comment:
Yes. I myself changed it to "No" some time age, when not all sources were downloadable this way and this step was failing. Now it works as expected.